### DISCUSSION TITLE
Add option to enable/disable tax rates and tax categories

### DISCUSSION THREAD

**Discussion by amphibian** [craftcms/commerce#3790] [Ideas] [answered]
Here in Canada we have an [upcoming two-month tax holiday](https://www.canada.ca/en/services/taxes/child-and-family-benefits/gst-hst-holiday-tax-break.html) (December 14-February 15) for certain categories of products, which is leading to online stores needing to figure out the best way to stop collecting taxes on these items for a short time.

Unfortunately, Craft Commerce currently has no way of temporarily disabling tax rates or tax categories aside from deleting them completely, meaning we'd need to recreate them all manually at midnight on February 16th.

**There would be more than one grateful developer in Canada if you could let us manage enabled/disabled status on tax rates and tax categories before December 14, 2024!**

**Comment by lukeholder** [comment]
@amphibian Commerce 5?

**Reply by amphibian** [reply]
The site in question for my client is on Commerce 4 - it didn't occur to me that this may have already been addressed in Commerce 5 though. Would upgrading to Commerce 5 resolve this for me?

**Reply by lukeholder** [reply, marked as answer]
We have added an enabled option to tax rates (but not categories) for the next release of Commerce (should be 5.3) https://github.com/craftcms/commerce/pull/3798

Unfortunately, we are not adding new features like that to previous versions of Commerce unless it solves a bug.

I would recommend making a content migration in your project that edits appropriate tax rates to set them to a zero rate, and a second migration prepared that restores that to the original rate, this would allow you to deploy the first migration to turn them off, and the second migration to turn them all on again at one time.

Sorry I cant be more help.

**Comment by nfourtythree** [comment]
Commerce 5.3 has now been released with this included.

Thanks!
